uwblueprint/childrens-aid-society
# add input styling

## Notion ticket link

[Setup Input Styling](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* added a new input style file and followed the figma to match styling



## Steps to test
1. pull from the branch
2. use the input at / to see if it matches figma designs
<img width="1054" alt="Screen Shot 2022-08-07 at 6 32 14 PM" src="https://user-images.githubusercontent.com/77185514/183313541-f663360a-43db-4ac1-b781-97e3c84674f4.png">



## What should reviewers focus on?
* checking if the styling is correct


## Checklist
